internal/configs/virtualserver_test.go

```diff
@@ -1960,3 +1961,44 @@ func TestGenerateEndpointsForUpstream(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateSlowStartForPlusWithInCompatibleLBMethods(t *testing.T) {
+	name := "test-slowstart-with-incompatible-LBMethods"
+	upstream := conf_v1alpha1.Upstream{Service: name, Port: 80, SlowStart: "10s"}
+	expected := ""
+
+	for lbMethod, _ := range incompatibleLBMethodsForSlowStart {
+		nlbMethod := generateLBMethod(upstream.LBMethod, lbMethod)
```

Comment:
there is no need to call `generateLBMethod`. this is the same comment as for line 1998.